tests/aws/services/lambda_/event_source_mapping/test_lambda_integration_sqs.py

```diff
@@ -1042,6 +1043,182 @@ def test_sqs_event_source_mapping(
         rs = aws_client.sqs.receive_message(QueueUrl=queue_url_1)
         assert rs.get("Messages", []) == []
 
+    @pytest.mark.parametrize("batch_size", [15, 100, 1_000, 10_000])
```

Comment:
nice coverage 💯 

they took ~45s on my M1, but could be slower after we implement the window mechanism, hence it's good to adjust the window for LocalStack here 💡